HCore-Audit/Databasae/Facades/IAuditFacade.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using HCore.Audit.Models;
+using Microsoft.EntityFrameworkCore.ChangeTracking;
+
+namespace HCore.Audit.Databasae.Facades
+{
+    public interface IAuditFacade
+    {
+        Task LogChangesAsync(ChangeTracker changeTracker, CancellationToken cancellationToken = default);
```

Comment:
I think this is too generic - events should be logged in a controlled manner, not flat across all things happening in the system